prime-router/src/main/kotlin/cli/CommandUtilities.kt

```diff
@@ -0,0 +1,57 @@
+package gov.cdc.prime.router.cli
+
+import com.github.kittinunf.fuel.Fuel
+import com.github.kittinunf.fuel.core.extensions.authentication
+import com.github.kittinunf.result.Result
+import gov.cdc.prime.router.common.Environment
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.runBlocking
+import java.io.IOException
+import java.net.URL
+
+/**
+ * Utilities for commands.
+ */
+class CommandUtilities {
+    companion object {
+        /**
+         * The API endpoint to check for that supports the HEAD operation.
+         */
+        private const val waitForApiEndpointPath = "/api/settings/organizations"
```

Comment:
This looks like it's using the same path to wait for the api regardless of what's calling it, but this will never pass for the lookup table call because this api needs the lookup tables to be populated.

Reply:
Yes, this will pass as the API will return an empty list.  For this purpose, any good return including an empty list works perfectly.

Reply:
looks good with the change.